tests/test_cli.py

```diff
@@ -140,11 +140,12 @@ def test_aggregations_process(script_info, event_queues, es, indexed_events):
     assert result.exit_code == 0
 
     es.indices.refresh(index='*')
-    assert agg_alias.count() == 54
-    assert agg_alias.doc_type('file-download-agg-bookmark').count() == 8
+    # import wdb; wdb.set_trace()
+    assert agg_alias.count() == 46
```

Comment:
Theoretically, all these numbers should remain the same in terms of grand totals (i.e. # of bookmark and # of aggregation docs)